resolwe/flow/managers/stat_utils.py

```diff
@@ -0,0 +1,162 @@
+"""Statistical utilities for the listener."""
```

Comment:
I'm not sure `resolwe.flow.managers` is the best place for such a static utility module.

@dblenkus, you probably have the best overview of the project's directory structure. Where should this reside?

Reply:
You can put it in `resolwe/flow/utils/`.